### PR TITLE
High level FxA sdks documentation

### PR DESCRIPTION
**Pull request by carolkng**
Fixes https://github.com/mozilla/application-services/issues/127
Based on the ios docs from #80 
Related to https://github.com/mozilla-mobile/android-components/issues/426

ping @eoger @vladikoff

### CODE REVIEW

File: docs/accounts/fxa-client-android.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+See Firefox Account documentation for that. 
+
+Once you have the OAuth info, you can start adding `FxAClient` to your Android project.
+As part of the OAuth flow your application will be opening up a Web view or open the system browser.
+Currently the SDK does not provide the Web view, you have to write it yourself.
+
+We use JCenter to distribute this library. Add the following to your `build.gradle`:
+
+```
+compile "org.mozilla.components:account:0.15"
```
Comment (vladikoff) [inline docs/accounts/fxa-client-android.md]: `:fxa:0.15`

File: docs/accounts/fxa-client-android.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+---
+id: account-client-android
+title: Android SDK
+---
+
+The **Firefox Accounts Android SDK** provides a way for Android applications to do the following:
```
Comment (vladikoff) [inline docs/accounts/fxa-client-android.md]: let's mirror this on https://github.com/mozilla-mobile/android-components/tree/master/components/service/firefox-accounts#setting-up-the-dependency

Comment (vladikoff) [reply docs/accounts/fxa-client-android.md]: the sample-fxa app can describe some of the functionality, custom tab, vs webview, vs wantsKeys...

**Comment by carolkng** [comment]
The a-c docs have been merged, so I think this can also be merged without fear of breaking links

**Comment by eoger** [comment]
Looks like this needs a rebase

**Comment by carolkng** [comment]
rebased

**Comment by eoger** [comment]
Great work thank you!

**Pull request merged**